### ISSUE TITLE
HBlock

### ISSUE THREAD

**Issue by ShadowGaming100** [AdguardTeam/HostlistsRegistry#549]
### Prerequisites

- [X] The blocklist is created specifically for DNS-level blockers;
- [X] The blocklist is regularly updated (at least 10 updates per month);
- [X] The blocklist is known and popular;
- [X] The blocklist satisfies [the repo requirements](https://github.com/AdguardTeam/HostlistsRegistry#-what-blocklists-can-be-added-here);

### Blocklist name

HBlock

### Blocklist category

Ad blocking, Trackers blocking, Malware&phishing blocking, Other

### Homepage

https://hblock.molinero.dev/

### Explanation

hBlock is a POSIX-compliant shell script that gets a list of domains that serve ads, tracking scripts and malware from [multiple sources](https://github.com/hectorm/hblock/blob/master/SOURCES.md) and creates a [hosts file](https://en.wikipedia.org/wiki/Hosts_(file)), [among other formats](https://github.com/hectorm/hblock/blob/master/resources/alt-formats), that prevents your system from connecting to them

**Comment by ShadowGaming100** [comment]
i saw the old issue, but it should be added as it helps. User will have to wait less time to search the blocklists as they know the blocklist that they want is already available. User will not needed to search the sub blocklists

**Comment by ShadowGaming100** [comment]
Not all blocklists that make up hblock is in dashboard so, please add the filter, it will be very helpful

**Comment by ghymacx** [comment]
nextdns has it also

**Comment by mikropsoft** [comment]
Must be added!!

**Comment by mikropsoft** [comment]
Must be added!!

**Comment by ShadowGaming100** [comment]
i can create a pull request if is agreed

**Comment by KnightmareVIIVIIXC** [comment]
https://github.com/AdguardTeam/HostlistsRegistry/issues/544

https://github.com/AdguardTeam/HostlistsRegistry/issues/506

https://github.com/AdguardTeam/HostlistsRegistry/issues/442

**Comment by Linux-Is-Best** [comment]
hBlock is why I use Next DNS over AdGuard DNS.

**Comment by KnightmareVIIVIIXC** [comment]
Based on the sources used, it seems like you could just build hBlock with what is offered in AdGuard already.
https://github.com/hectorm/hblock/blob/master/SOURCES.md

**Comment by Linux-Is-Best** [comment]
> Based on the sources used, it seems like you could just build hBlock with what is offered in AdGuard already. https://github.com/hectorm/hblock/blob/master/SOURCES.md

No, you cannot.  You're missing more than a few from that list.

**Comment by mikropsoft** [comment]
> Based on the sources used, it seems like you could just build hBlock with what is offered in AdGuard already.
> https://github.com/hectorm/hblock/blob/master/SOURCES.md

Since we don't want to add them one by one, we use these lists. Hagezi works the same way

**Comment by Linux-Is-Best** [comment]
> > Based on the sources used, it seems like you could just build hBlock with what is offered in AdGuard already.
> > https://github.com/hectorm/hblock/blob/master/SOURCES.md
> 
> Since we don't want to add them one by one, we use these lists. Hagezi works the same way

Exactly.  I don't know why they seem to against, hBlock.

**Comment by ShadowGaming100** [comment]
Issue similar to this has been closed 

There are people needing this blocklist so please add it

**Comment by ShadowGaming100** [comment]
> > > Based on the sources used, it seems like you could just build hBlock with what is offered in AdGuard already.
> > > https://github.com/hectorm/hblock/blob/master/SOURCES.md
> > 
> > 
> > Since we don't want to add them one by one, we use these lists. Hagezi works the same way
> 
> Exactly. I don't know why they seem so against, hBlock.

Agree

**Comment by KnightmareVIIVIIXC** [comment]
> > > Based on the sources used, it seems like you could just build hBlock with what is offered in AdGuard already.
> > > https://github.com/hectorm/hblock/blob/master/SOURCES.md
> > 
> > 
> > Since we don't want to add them one by one, we use these lists. Hagezi works the same way
> 
> Exactly. I don't know why they seem so against, hBlock.

Probably for the reason I mentioned: AdGuard has the sources used by hBlock and the EasyList and uBlock lists that AdGuard doesn't have weren't built to be DNS blocklists in the first place. Other than that, you can definitely build hBlock by enabling OISD, Steven Black, 1Hosts Lite, and Hagezi Ultimate.

However, I will say that it would be wise to add compilation lists because that would be the easiest way to add multiple blocklists without having to add them separately.

**Comment by Linux-Is-Best** [comment]
> > > > Based on the sources used, it seems like you could just build hBlock with what is offered in AdGuard already.
> > > > https://github.com/hectorm/hblock/blob/master/SOURCES.md
> > > 
> > > 
> > > Since we don't want to add them one by one, we use these lists. Hagezi works the same way
> > 
> > 
> > Exactly. I don't know why they seem so against, hBlock.
> 
> Probably for the reason I mentioned: AdGuard has the sources used by hBlock and the EasyList and uBlock lists that AdGuard doesn't have weren't built to be DNS blocklists in the first place. Other than that, you can definitely build hBlock by enabling OISD, Steven Black, 1Hosts Lite, and Hagezi Ultimate.
> 
> However, I will say that it would be wise to add compilation lists because that would be the easiest way to add multiple blocklists without having to add them separately.

And as previously pointed out, that does not include everything hBlock does.

**Comment by KnightmareVIIVIIXC** [comment]
I don't think hBlock will function the same way after it's compressed and validated.

**Comment by Linux-Is-Best** [comment]
> I don't think hBlock will function the same way after it's compressed and validated.

Does it block AdGuard or something? Because this excuse seems like nonsense.

**Comment by KnightmareVIIVIIXC** [comment]
https://github.com/AdguardTeam/HostlistsRegistry?tab=readme-ov-file#-what-blocklists-can-be-added-here

`We prefer (mostly) original lists to compilations.`

**Comment by Linux-Is-Best** [comment]
> https://github.com/AdguardTeam/HostlistsRegistry?tab=readme-ov-file#-what-blocklists-can-be-added-here
> 
> `We prefer (mostly) original lists to compilations.`

And what makes this list different than your other compilations that you do include? As repeatedly noted, hBlock includes sources you do not. That makes hBlock needed. So I ask again, why the hate for hBlock?

**Comment by ShadowGaming100** [comment]
What is the plan will this be added??

**Comment by KnightmareVIIVIIXC** [comment]
> The missing sources are already covered by other lists like OISD or Hagezi, so you should use those

Did you just flip your opinion midway or am I crazy?

**Comment by Sergey-Lyapin** [comment]
Thanks for the suggestion.
This list is a compilation, and we prefer original lists to compilations.
We recommend using Adguard DNS filter or/and other available lists.
